cmd/titus-log-viewer/main_test.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestPing(t *testing.T) {
```

Comment:
What is this test actually testing? Looks like it's just using golang http packages.

Reply:
honest I don't know.  I just cloned the test from darion.